baseband/helpers/tests/test_sequentialfile.py

```diff
@@ -288,3 +291,42 @@ def test_memmap_fail(self, tmpdir):
         with sf.open(self.files, 'wb', file_size=10) as fh:
             with pytest.raises(ValueError):
                 fh.memmap(shape=(5,))
+
+
+class TestFileNameSequencer(object):
+    def setup(self):
+        with open(SAMPLE_FILE, 'rb') as fh:
+            self.header = vdif.VDIFHeader.fromfile(fh)
+
+    def test_enumeration(self):
+        fns1 = sf.FileNameSequencer('x{file_nr:03d}.vdif', {})
+        assert fns1[0] == 'x000.vdif'
+        assert fns1[107] == 'x107.vdif'
+        fns2 = sf.FileNameSequencer('{SNAKE}_{file_nr}', {'SNAKE': 'python'})
+        assert fns2[13] == 'python_13'
+
+        with pytest.raises(KeyError):
+            sf.FileNameSequencer('{SNAKE:06d}.x', {'PYTHON': 10})
+
+    def test_header_extraction(self):
+        # Follow the typical naming scheme:
+        # 2016-04-23-07:29:30_0000000000000000.000000.dada
```

Comment:
Why have a `dada` file example for creating a `vdif` one? I'd go with an ARO vdif example (which are stored in directories with names, so possibly include that?)

Reply:
That was a vestigial comment because I copy-pasted the tests from DADA.  I've removed it, and also checked the rest of the PR for outdated comments.